packages/vscode-extension/src/envTree.ts

```diff
@@ -66,16 +66,21 @@ export async function registerEnvTreeHandler(
       for (const item of envNames) {
         showEnvList.push(item);
         const provisionSucceeded = await getProvisionSucceedFromEnv(item);
+        const isLocal = item === LocalEnvironmentName;
         environmentTreeProvider.add([
           {
             commandId: "fx-extension.environment." + item,
             label: item,
             description: provisionSucceeded ? "(Provisioned)" : "",
             parent: TreeCategory.Environment,
-            contextValue: item === LocalEnvironmentName ? "local" : "environment",
+            contextValue: isLocal
```

Comment:
Will this get updated after user provision?

Reply:
Yes, the treeview will update after provision.